code/modules/mob/living/carbon/species.dm

```diff
@@ -962,4 +967,125 @@ var/global/list/whitelisted_species = list("Human")
 	brute_mod = 0.8
 	move_speed_multiplier = 2
 
-	blood_color = "#7FFF00"
\ No newline at end of file
+	blood_color = "#7FFF00"
+
+/datum/species/slime
+	name = "Slime"
+	icobase = 'icons/mob/human_races/r_slime.dmi'
+	deform = 'icons/mob/human_races/r_def_slime.dmi'
+	known_languages = list(LANGUAGE_SLIME)
+	meat_type = /obj/item/slime_heart
+	attack_verb = "glomps"
+
+	flags = IS_WHITELISTED | NO_BREATHE
+	anatomy_flags = NO_SKIN | NO_BLOOD | NO_BONES | NO_STRUCTURE | MULTICOLOR
+
+	spells = list(/spell/regen_limbs)
+
+	gender = NEUTER
+
+	tox_mod = 2
+
+	primitive = /mob/living/carbon/slime/pygmy
+
+	blood_color = "#96FFC5"
+	flesh_color = "#96FFC5"
+
+	cold_level_1 = 273  // Cold damage level 1 below this point.
+	cold_level_2 = 250  // Cold damage level 2 below this point.
+	cold_level_3 = 230  // Cold damage level 3 below this point.
+
+	heat_level_1 = 330  // Heat damage level 1 above this point.
+	heat_level_2 = 350  // Heat damage level 2 above this point.
+	heat_level_3 = 373  // Heat damage level 3 above this point.
+
+	has_mutant_race = 0
+
+	has_organ = list(
+		"brain" =    /datum/organ/internal/brain/slime_core,
+		)
+
+/datum/species/slime/xenobio
+	name = "Evolved Slime"
+	flags = IS_WHITELISTED | NO_BREATHE | ELECTRIC_HEAL
+
+/datum/species/slime/handle_death(var/mob/living/carbon/human/H) //Handles any species-specific death events (such as dionaea nymph spawns).
+	if(!isslimeperson(H))
```

Comment:
Needlessly defensive. If this ever actually happens, I'd much rather hear about it than have it be caught silently.